imports/ui/components/FreshForm.jsx

```diff
@@ -33,15 +38,28 @@ class FreshForm extends Component {
     this.setState({ price: event.target.value });
   };
 
+  setAddress(address) {
+    this.setState({
+      add: address
+    });
+  }
+  setLatLng(latLng) {
+    this.setState({
+      lat: latLng.lat,
+      lng: latLng.lng
+    });
+  }
+
   handleSubmit = event => {
     let newItem = {
       item: this.state.item,
       price: "$" + this.state.price,
       uuid: uuidv4(),
       location: {
+        address: this.state.add,
```

Comment:
Can you also add word wrap to the address? 

![Screen Shot 2019-06-26 at 1 43 13 PM](https://user-images.githubusercontent.com/25490855/60213522-8afd3d80-9818-11e9-89ce-270e20edb44a.png)

Reply:
> That brings up another point, maybe for a different PR. Can you have the ability to add "Other" where the user then gets to input what they want? Beats having us coming up with the list.

Oh I thought the selectBar was the only options we were going to give?? Otherwise, the item field we have now should be enough o:

Reply:
Resolved the word-wrap